sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureServiceClientOptions } from '@azure/core-arm';
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptions extends AzureServiceClientOptions {
+    // (undocumented)
+    baseUri?: string;
+}
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.RequestOptionsBase {
+    models?: any[];
```

Comment:
`any` isn't very useful - should this be genericized to use a template parameter?

Reply:
That s how the service defines it. Users can create their own objects, relations. 

---
In reply to: [447255604](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447255604) [](ancestors = 447255604)